docs/developer-guide/contributing.md

```diff
@@ -156,6 +156,29 @@ make: *** [Makefile:386: verify-kube-connect] Error 1
 
 you should edit your `~/.kube/config` and modify the `server` option to point to your correct K8s API (as described above).
 
+### Using k3d
+
+k3d is a lightweight wrapper to run [k3s](https://github.com/rancher/k3s), a minimal Kubernetes distribution, in docker. Because it's running in a docker container, you're dealing with docker's internal networking rules when using k3d. A typical Kubernetes cluster running on your local machine is part of the same network that you're on so you can access it using **kubectl**. However, a Kubernetes cluster running within a docker container (in this case, the one launched by make) cannot access 0.0.0.0 from inside the container itself, when 0.0.0.0 is a network resource outside the container itself (and/or the container's network). This is the cost of a fully self-contained, disposable Kubernetes cluster. The following steps should help with a successful `make verify-kube-connect` execution.
+
+1. Find the actual external IP of your host. You can do so by executing `ifconfig` on Mac/Linux and `ipconfig` on Windows. For most users, the following command works to find the IP address.
+
+```
+IP=`ifconfig en0 | grep inet | grep -v inet6 | awk '{print $2}'`
```

Comment:
It will error out on linux and windows because the same interface device won't be available

Reply:
on linux you can run `ip addr`. I don't know about windows.

Reply:
Also keep in mind this IP will be dynamically assigned by the router so if your router restarts for any reason you will have to recreate the cluster. You can assign your PC a static IP to prevent that from happening

Reply:
Thanks for the review Amit. I added command for Linux.